lib/pure/parseutils.nim

```diff
@@ -592,6 +592,71 @@ proc parseFloat*(s: openArray[char], number: var float): int {.
   if result != 0:
     number = bf
 
+func toLowerAscii(c: char): char =
+  if c in {'A'..'Z'}: char(uint8(c) xor 0b0010_0000'u8) else: c
+
+func parseSize*(s: openArray[char], size: var int64, alwaysBin=false): int =
```

Comment:
Considering is totally new, should it use `size: out int64` ?. 🤔

Reply:
I am kind of hoping to backport it to nim-1.6..

Reply:
Also, all of the other `parseFoo` in the module still just use a `var`... So, there may be a "change them all at the same time" kind of  consistency principle in play.